dev-python/colorcet/metadata.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>sci@gentoo.org</email>
+		<name>Gentoo Science Project</name>
+	</maintainer>
+</pkgmetadata>
```

Comment:
Could you [add](https://devmanual.gentoo.org/keywording/index.html#simultaneous-stabilization-on-all-architectures) ```<stabilize-allarches/>``` if it's pure Python (no compilation and so on)?

(Applies to all packages where appropriate, as with most of these comments - I'l try make clear if it's just one.)

Reply:
**Nice!** I'll do so. Shamefully, this is the first I've heard of `<stabilize-allarches/>`. I will now use that everywhere.

There's an interesting parallel here with `conda-forge`, the automated packaging system for third-party Anaconda packages. `conda-forge` recipes support a [similar `noarch: python` directive](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-builds):

> Noarch packages are packages that are not architecture specific and therefore only have to be built once. Declaring these packages as `noarch` in the `build` section of the `meta.yaml` reduces shared CI resources. Therefore all packages that qualify to be `noarch packages`, should be declared as such.